server/admin/admin_server.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package admin
+
+import (
+	"github.com/golang/protobuf/ptypes/empty"
+	"github.com/google/trillian"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+)
+
+var errNotImplemented = grpc.Errorf(codes.Unimplemented, "not implemented")
+
+// adminServer is an implementation of trillian.TrillianAdminServer.
+type adminServer struct{}
+
+// New returns a trillian.TrillianAdminServer implementation.
+func New() trillian.TrillianAdminServer {
+	return &adminServer{}
```

Comment:
Yes. Here and pretty much everywhere else on the PR.

At the very least this will need to be `func New(extension.Registry)`.